dev/io.openliberty.netty.internal.tls.impl/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Liberty Netty Transport TLS Provider Implementation
+Bundle-SymbolicName: io.openliberty.netty.internal.tls.impl
+Bundle-Description: Liberty Netty Transport TLS Provider Implementation ${bVersion}
+WS-TraceGroup: Netty
+
+Export-Package: io.openliberty.netty.internal.tls.impl
+
+Import-Package: \
+  io.openliberty.netty.internal.tls,\
+  ${defaultPackageImport}
+
+-dsannotations: \
+  io.openliberty.netty.internal.tls.impl.NettyTlsProviderImpl
+
+-buildpath: \
+  com.ibm.ws.logging,\
+  com.ibm.websphere.org.osgi.service.component,\
+  com.ibm.wsspi.org.osgi.service.component.annotations,\
+  com.ibm.ws.ssl, \
+  com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+  io.openliberty.io.netty;version=latest,\
+  io.openliberty.io.netty.ssl;version=latest,\
+  io.openliberty.netty.internal;version=latest,\
```

Comment:
I believe you can get rid of this trailing ,\ on the last line

Reply:
done